documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
+
+
+### Allocating Resources to Projects
+The Corporate Overlords can vote to assign budgets to BIFFUD Projects
+which are then overseen by the project owners.  The project owners
+must select a single person to be the project's "Cashier" who is
+responsible for approving any invoices associated with a project or
+working with the BIFFUD Treasurer to make purchases using BIFFUD funds.
+
+At the time of budget approval the Corporate Overlords must specify if
+the resources are being "given" or "loaned" to the project.
+
+Invoices that are not approved by a project's Cashier will not be compensated
+unless they are explicitly approved by the BIFFUD Treasurer.
+
+
+### Distributing Profits
+Profits are to be calculated at the end of the fiscal year.  If by
+some miracle there are profits on a given year, the following formula
+will dictate its distribution
+
+1. A mandatory distribution of profit will be made to offset individual
+taxes on that profit.  Corporate Overlords may choose to prevent this
+distribution my making a 70% vote during a plotting session.
+
+2. 60% of the remaining funds will be distributed to the projects
+in proportion to the project-level profit distribution.
+
+2b. If a project has a profit allocation and the project had been allocated
+a "loan" from BIFFUD, that project's allocated profit is reduced by BIFFUD
+and subtracted from the outstanding balance.
+
+2c. The project owners may vote to decide to either distribute the profits
+proportionally according to their percentage ownership of the project, or
+to allocate the profit towards future project expenses.
+
+3. The remaining 40% of the profits is kept by BIFFUD.
+
+3b. 50% of the remaining BIFFUD profits is to be distributed to the Corporate
+Overlords in terms of their ownership percentage.  The remaining 50% is to
+be kept as corporate funds.  The Corporate Overlords may adjust this ratio
+for a given year during a monthly plotting session.
+
+
+### Compensation
+Any compensation paid to individuals working on a project must be associated
+with a contract that has either been approved by the Treasurer or by the
+Corporate Overlords during a plotting session.
+
+
+### Expenses
+Members and Coporate Overlords may receive reimbursement for expenses associated
+with a project through the processes outlined in the "Allocating Resources For
+Projects" section.
+
+Expenses under $100.00 may be approved by the Treasurer at any time or can be
+approved by Corporate Overlords during a plotting session.  All other expenses
+must be approved by Corporate Overlords during a plotting session.
+
+
+### Company Canoe
+During the plotting session at the end of each fiscal year the Corporate Overlords
+must consider a vote on the purchase of a company canoe, to be named and labeled the
+"BIFFUD Canoe".  If ever approved, the Corporate Overlords must update this agreement
+with a record of the achieved milestone as well as the identification of a new
+capitalistic goal.
```

Comment:
First, I can't believe this comment is going in a section about the company canoe (happy disbelief, mind you). But here we are. How are we determining what our fiscal year is? Different from the CO term limit year? Calendar year? Hebrew year?